python/BioSimSpace/Sandpit/Exscientia/Align/_alch_ion.py

```diff
@@ -0,0 +1,53 @@
+from .._SireWrappers import Molecule as _Molecule
+from .._Exceptions import IncompatibleError as _IncompatibleError
+
+
+def _mark_alchemical_ion(molecule):
```

Comment:
Should this be private?

Reply:
So in the later stage, this function will be executed as part of `fix_perturbed_charge`, which will be the public function for user to use.

Reply:
Ah so this will be in the public repo now?